### DISCUSSION TITLE
Migrate from angular.json to project.json

### DISCUSSION THREAD

**Discussion by mehrad-rafigh** [nrwl/nx#11015] [General]
Hi there,

is there a possibility to migrate from angular.json to project.json? I couldn't find any documentation on that.

**Comment by leosvelperez** [comment]
We have a generator for that: https://nx.dev/packages/workspace/generators/convert-to-nx-project.

Make sure your `angular.json` version is set to 2 and run `nx g @nrwl/workspace:convert-to-nx-project --all`.

**Reply by mehrad-rafigh** [reply]
Awesome @leosvelperez!

I will give a try

**Reply by viceice** [reply]
new docs link: https://nx.dev/packages/workspace/generators/fix-configuration
